### PR TITLE
Log Improvements

### PR DESCRIPTION
**Pull request by Ashod**
- wsd: clang-tidy: suppress bogus assert diagnostics
- wsd: http: correct written data-size reporting
- wsd: ssl: log the operation context in errors
- wsd: correct logged upload duration
- wsd: improve various logs

### CODE REVIEW

File: net/WebSocketSession.hpp [hunk 1]
```diff
@@ -343,8 +343,6 @@ class WebSocketSession final : public WebSocketHandler
 
             if (_shutdown && _outQueue.isEmpty())
             {
-                LOG_DBG(
-                    "WebSocketSession: closing gracefully after sending all the data, as flagged.");
```
Comment (eszkadev) [inline net/WebSocketSession.hpp:347]: just to confirm: we don't want this log?

File: wsd/DocumentBroker.cpp [hunk 2]
```diff
@@ -1266,13 +1269,19 @@ void DocumentBroker::handleSaveResponse(const std::string& sessionId, bool succe
...
+    // Rename even if no new save, in case we have an older version.
     if (rename(oldName.c_str(), newName.c_str()) < 0)
     {
         // It's not an error if there was no file to rename, when the document isn't modified.
         const auto onrre = errno;
-        LOG_DBG("Failed to renamed [" << oldName << "] to [" << newName << "] ("
-                                      << Util::symbolicErrno(onrre) << ": " << std::strerror(onrre)
-                                      << ')');
+        if (success || onrre != ENOENT)
+            LOG_ERR("Failed to rename [" << oldName << "] to [" << newName << "] ("
```
Comment (eszkadev) [inline wsd/DocumentBroker.cpp:1278]: I guess you wanted to print something different here than the else block will :)

**Review by eszkadev — APPROVED** [review]
added 2 comments, but looks good

File: wsd/DocumentBroker.cpp (hunk 2 above)
Comment (Ashod) [reply wsd/DocumentBroker.cpp:1278]: The log-level is different. I could make it more complex to reduce duplication, but the number of lines and readability will probably not improve.

I thought of adding more info, or to differentiate, but for some reason couldn't find anything to change. So I left them identical.

**Pull request merged**